src/views/HomeView.vue

```diff
@@ -19,25 +21,33 @@ const correctAnswer = ref("");
 const answeredQuestion = ref(false);
 const answeredCorrectly = ref(false);
 
-
-function checkAnswer(answer){
+const checkAnswer = (answer) => {
```

Comment:
Nice catching this! Let's continue using arrow fuctions in the fucture like we decided in planning